StreamVideoTests/WebRTC/Retries_Tests.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamVideo
+import WebRTC
+import XCTest
+
+final class Retries_Tests: XCTestCase {
+    
+    let dummyRequest = URLRequest(url: URL(string: "https://test.com")!)
+    let dummyData = Data()
+    let dummyError = ClientError.NetworkError()
+    var dummyState = "dummy"
```

Comment:
```suggestion
    private var dummyRequest: URLRequest! = URLRequest(url: URL(string: "https://test.com")!)
    private var dummyData: Data! = Data()
    private var dummyError: Error! = ClientError.NetworkError()
    private var dummyState: String! = "dummy"
    
    func tearDown() throws async {
        dummyRequest = nil
        dummyData = nil
        dummyError = nil
        dummyState = nil
        try await super.tearDown()
    }
```

or make them private static